### PR TITLE
Ft/UI update

### PR DESCRIPTION
**Pull request by kelvinkipruto**
Update page ui based on [design](https://www.figma.com/file/0eTeMRUvs87QFm0QdO3xOM/CfA-Jigsaw-Harrasmentmanger?node-id=13%3A656)

### CODE REVIEW

**Comment by kilemensi** [comment]
`npm run build:all:dev` fails on my end @kelvinkipruto.

**Comment by kelvinkipruto** [comment]
@kilemensi Let me have a look.

**Comment by esirK** [comment]
👍  @kelvinkipruto 
On my machine, 
Screenshot 2022-09-22 at 13 52 26

1. There is an Outline around the `Safe Net` title which I can't see on the design.
2. The `FAQs` are already opened 
3. 
Screenshot 2022-09-22 at 13 58 38

The outline looks a little bit off for me.

**Comment by kelvinkipruto** [comment]
@esirK Thanks. Looking into this also.

**Review by esirK — APPROVED** [review]
Looks good @kelvinkipruto just one small thing. The drop-down buttons aren't visible maybe use the same color as the text?

Screenshot 2022-09-26 at 11 53 34

File: src/styles.scss [hunk 1]
```diff
@@ -127,17 +127,18 @@ body {
 app-help-center
   .mat-form-field-appearance-outline
   .mat-form-field-outline-start {
-  border-radius: 30px 0 0 30px;
```
[the remark this answers is not in the document]
Comment (kilemensi) [reply src/styles.scss:130]: Correct fonts are still not loading @kelvinkipruto . I don't think you've included all changes suggested here e.g.
```scss
@import url('//fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Merriweather:wght@400;700&family=Roboto&display=swap');
```
is missing.

[the remark this answers is not in the document]
Comment (kelvinkipruto) [reply src/styles.scss:130]: @kilemensi Updated.

**Review by kilemensi — APPROVED** [review]
👍🏽 @kelvinkipruto 

----

Lets merge as is and review page by page, component by component after... Issues like 👇🏽 are in almost all pages/components so reviewing/fixing them in one go seem like a challenge.

Screenshot from 2022-09-27 12-53-00

**Pull request merged**